mirrord/config/src/feature/copy_target.rs

```diff
@@ -7,6 +7,17 @@ use crate::{
     util::MirrordToggleableConfig,
 };
 
+/// Default value for [`CopyTargetFileConfig::enabled`] when deserializing.
+/// This way in [`MirrordConfig::generate_config`] we can make distinction between following two
+/// cases:
+/// 1. `{ "features": {} }` - copy target is disabled, we get `enabled: false` from [`Default`]
+///    implemetation.
+/// 2. `{ "features": { "copy_target": {} } }` - copy target is enabled, we get `enabled: true` from
```

Comment:
This is not the nicest interface to enable a feature. I think it might be a bit weird for users.

I think we could make it take a single value (not an object) and it would be a nicer interface.
Something like:
`true`/`"on"`/`"copy"` -> copy without scaling down.
`"replace"`/`"scale-down"` -> copy with scaling down.
`false`/`"off"`/not specified -> no copy

What do you think?

Either way, doesn't have to be in this PR since the feature is marked as unstable so we can change the interface again later.

Reply:
I think eventually we'll have more customization here. Right now the user can use all of `true`, `false`, `{}`, `{"scale_down": ...}` or omit this config entirely; so I think it follows the pattern of other configs

Reply:
Oh, if they can just use `true` that's fine. Does `"copy_target": true` mean copy without scaling down?